### ISSUE TITLE
Massiv, Acclerate, (and eventually Hasktorch)

### ISSUE THREAD

**Issue by o1lo01ol1o** [Magalame/fastest-matrices#2]
Hey, cool benchmarks!  It'd be nice to also see benchmarks for the above since they also offer promising numerical performance.

**Comment by Magalame** [comment]
Hi! Thanks :)
Good point, I was planning to add `massiv` in particular. The thing about `accelerate` is that it might make the replication process a bit more cumbersome, particularly since `accelerate` relies on `libFFI`, and on my machine I've encountered this issue: https://gitlab.haskell.org/ghc/ghc/issues/15397
I didn't know Hasktorch had its own linear algebra operations, thanks for letting me know!

**Comment by o1lo01ol1o** [comment]
Right, the llvm toolchain can be fiddly. We may have some nix expressions laying around, if so I’ll  open a PR; they’d make a good addition for reproducible benches anyway.

**Comment by Magalame** [comment]
Sounds good, thanks for taking care of this!
There is also a PR for reproducing builds with docker, if that helps you: https://github.com/Magalame/fastest-matrices/pull/1

**Comment by o1lo01ol1o** [comment]
As an update, we don't have anything in the moment, but I'll post back if / when we do.

**Comment by o1lo01ol1o** [comment]
As a reminder, `nixpkgs` doesn't currently provide a derivation for `llvm-8` which is required for ghc 8.6; accelerate expressions will be possible when they do.

**Comment by Magalame** [comment]
Oh I see. So I'll just add `massiv` for now
